schema/release-schema.json

```
@@ -1129,6 +1129,19 @@
             "description": "This field is deprecated in favor of `languages`, to support documents in which multiple languages are used.",
             "deprecatedVersion": "1.2"
           }
+        },
+        "relatedItems": {
+          "title": "Related items",
+          "description": "If the document relates to one or more items, the IDs of the related item(s) in the items array of the parent object. For example, for a document in `tender.documents`, the `.id` of the related item(s) in `tender.items`.",
```

The "if" clause is not needed, as this is true for any field. The "(s)" construction would have to be "the ID(s) of the related item(s)", but this is unnecessarily awkward to read. The plural is fine (like the description of `documents` itself). The semantics are more properly the related items, not their identifiers – their identifiers are instead just a technical (not semantic) detail.

An important distinction to make is that this field is only relevant if the document applies to a subset of items.

```suggestion
          "description": "The subset of items to which this document relates, expressed as the `id` values of the related entries in the parent object's `items` array. For example, an entry in the `tender.documents` array may be related only to entries in the `tender.items` array.",
```